### PR TITLE
Rabbitfix

### PR DESCRIPTION
**Pull request by drblgn**
**What kind of change does this PR introduce?**

Fixes vidcloud extractor.

**Did you add tests for your changes?**

I added a vidcloud_test.ts for my changes.

**Summary**

flixhq has been unusuable for sometime because the vidcloud extractor wasn't updated. this pr should hopefully fix it.

**Other information**

### CODE REVIEW

File: src/extractors/decodedpng.js
Comment (riimuru) [inline src/extractors/decodedpng.js:1]: Is this subject to change? Also, the file should be moved into rabbit.ts file

File: src/extractors/rabbit.ts [hunk 1]
```diff
@@ -0,0 +1,599 @@
+const data = require('./decodedpng.js');
+const cryptoJs = require('crypto-js');
```
Comment (riimuru) [inline src/extractors/rabbit.ts:2]: use `import` instead of `require`

**Review by riimuru — APPROVED** [review]

File: src/extractors/decodedpng.js
Comment (drblgn) [reply src/extractors/decodedpng.js:1]: nope not subject to change. Sure will move it to rabbit.ts

**Review by riimuru — APPROVED** [review]

**Comment by jairoxyz** [comment]
Rabbitfix is broken as of yesterday, I believe due to changes in the web assembly. First error is thrown because the embed page doesn't return any cookies anymore. Disabling getCookie then produces the next error when trying to instantiate the web assembly: [LinkError: WebAssembly.instantiate(): Import #71 module="wbg" function="__wbindgen_closure_wrapper101" error: function import requires a callable] So, obviously some code has changed in the wasm code in https://rabbitstream.net/images/loading.png?v=0.6. Thanks for the good work.

**Pull request merged**